opensearch_py_ml/ml_models/sentencetransformermodel.py

```diff
@@ -1086,6 +1098,65 @@ def make_model_config_json(
                 "all_config": json.dumps(all_config),
             },
         }
+
+        if pooling_mode is not None:
+            model_config_content["model_config"]["pooling_mode"] = pooling_mode
+        else:
+            pooling_config_json_file_path = os.path.join(
+                folder_path, "1_Pooling", "config.json"
+            )
+            if os.path.exists(pooling_config_json_file_path):
+                try:
+                    with open(pooling_config_json_file_path) as f:
+                        if verbose:
+                            print(
+                                "reading pooling config file from: "
+                                + pooling_config_json_file_path
+                            )
+                        pooling_config_content = json.load(f)
+                        if pooling_mode is None:
```

Comment:
Do we still need this check as line 1104 also doing the same I believe?

Reply:
You're right. I'll remove it.